src/Fixer/DoctrineAnnotation/DoctrineAnnotationIndentationFixer.php

```diff
@@ -72,7 +72,7 @@ protected function fixAnnotations(Tokens $tokens)
             if ($previousLineBracesDelta > 0) {
                 ++$indentLevel;
             }
-            if ($currentLineDelta < 0) {
+            if ($currentLineDelta < 0 && $indentLevel > 0) {
```

Comment:
btw, why indent level was < 0 ?

Reply:
Because on line 2 we have `(` and `{` that create a new indent level, but we ignore one as they are on the same line to avoid indenting next line twice. On line 3 we have `}` that removes one level, and on line 4 we have `)` that removes one level again... -1 :)